DrinkManagerWeb/Views/Shared/_Layout.cshtml

```diff
@@ -42,7 +42,7 @@
                     </li>
                     <li class="nav-item d-flex align-items-center mb-3 @(routeValues["Action"].ToString() == "Index" ? "active" : "")">
```

Comment:
In this line replace "Index" with "SearchByAlcoholContent".

Reply:
Done!